src/runtime/components/Slideover.vue

```diff
@@ -143,30 +143,30 @@ const ui = computed(() => tv({ extend: tv(theme), ...(appConfig.ui?.slideover ||
       >
         <VisuallyHidden v-if="!!slots.content && ((title || !!slots.title) || (description || !!slots.description))">
           <DialogTitle v-if="title || !!slots.title">
-            <slot name="title">
+            <slot name="title" :close="closeDialog">
               {{ title }}
             </slot>
           </DialogTitle>
 
           <DialogDescription v-if="description || !!slots.description">
-            <slot name="description">
+            <slot name="description" :close="closeDialog">
               {{ description }}
             </slot>
           </DialogDescription>
         </VisuallyHidden>
 
-        <slot name="content">
+        <slot name="content" :close="closeDialog">
           <div v-if="!!slots.header || (title || !!slots.title) || (description || !!slots.description) || (close || !!slots.close)" :class="ui.header({ class: props.ui?.header })">
-            <slot name="header">
+            <slot name="header" :close="closeDialog">
               <div :class="ui.wrapper({ class: props.ui?.wrapper })">
                 <DialogTitle v-if="title || !!slots.title" :class="ui.title({ class: props.ui?.title })">
-                  <slot name="title">
+                  <slot name="title" :close="closeDialog">
```

Comment:
Is it really necessary to pass the `close` method in the title and description? 🤔
```suggestion
                  <slot name="title">
```

Reply:
It's probably not going to be used I'd imagine, I added it just to be consistent to add it to all slots that don't already control the open state. I'll remove it, I don't think it will be used often and it can be added later if needed by someone.